modules/globebrowsing/src/layergroupid.cpp

```diff
@@ -31,66 +31,66 @@ namespace openspace::globebrowsing::layers {
 
 static_assert(
     std::is_sorted(
-        Groups.begin(),
-        Groups.end(),
```

Comment:
Just curious - is there some type of optimization/performance gain from using cbegin over begin? Due to the guaranteed constness?

Reply:
I did it for "its the right thing to do" reasons, exactly as you are saying. I was now curious as well and did some microbenchmarking, and here and there it does make a bit of a difference (depending on the run in either direction), the differences are purely academical.

Moving forward we should just use `.begin()` and slowly migrate over to this.  I'll change these examples back to how they were before

Reply:
Confused. Do you mean that we should use `.begin()` or `.cbegin()`?

Reply:
Using `.begin()` is easier to parse and read then `.cbegin()` and doesn't seem to make any difference in speed.  All microbenchmarks I did were coming up as them being the same